content/docs/releases/v1.1.0.md

```diff
@@ -0,0 +1,92 @@
++++
+title = "Release v1.1.0"
+short = "v1.1.0"
+weight = 1
+date = "2021-11-28"
++++
+
+NSM v1.1.1 is released via a set of [example use cases](https://github.com/networkservicemesh/deployments-k8s/tree/release/v1.1.1).
+
+## Kubernetes Compatibility
+NSM v1.1.1 has been successfully [integration tested](https://github.com/networkservicemesh/integration-k8s-kind/actions/runs/1514862151) with Kubernetes versions:
+
+- v1.22.1
+- v1.21.1
+- v1.20.7
+- v1.19.11
+- v1.18.15
+- v1.17.11
+
+## Known Issues
+
+###   GKE Compatibility
+GKE has manifest an odd Netlink IP Neighbor bug with IPv6 in its latest node-version.  This is expected to worked around automatically in NSM in the future.  Meanwhile, there is a workaround:
```

Comment:
Grammar here can be improved.

Reply:
Fixed in https://github.com/networkservicemesh/site/pull/207/commits/a94b37d7475128c89c4e040f6aac0789ff094afa